static/js/metrics-explorer.js

```diff
@@ -969,51 +977,29 @@ function initializeChart(canvas, seriesData, queryName, chartType) {
 }
 
 function addVisualizationContainer(queryName, seriesData, queryString) {
-     
-    var existingContainer = $(`.metrics-graph[data-query="${queryName}"]`)
+    var existingContainer = $(`.metrics-graph[data-query="${queryName}"]`);
     if (existingContainer.length === 0){
         var visualizationContainer = $(`
         <div class="metrics-graph" data-query="${queryName}">
             <div class="query-string">${queryString}</div>
             <div class="graph-canvas"></div>
         </div>`);
 
-        // Determine where to insert the new container
```

Comment:
Why are these codes removed? It broke some features on the metrics and alerts screen.